debian/clearwater-snmpd.postinst

```diff
@@ -58,6 +58,7 @@ case "$1" in
         cp /etc/snmp/snmpd.conf.clearwater-snmpd /etc/snmp/snmpd.conf
 
         # Create the folder to store the statistics UNIX domain sockets.
+        rm -rf /var/run/clearwater
```

Comment:
That's a shame. I can't accept this PR with this line in. Please can you remove it? 

I also strongly recommend that you **don't** just add this line back into Clearwater Core. If there is a problem here we need to understand and fix it properly, without deleting directories that other packages may be using.